images/server/install-sambacc.sh

```diff
@@ -34,7 +34,7 @@ case $action in
         container_json_file="/usr/local/share/sambacc/examples/minimal.json"
     ;;
     install-rpm)
-        dnf install -y "${rpmfiles[0]}"
+        dnf install --skip-broken --nobes -y "${rpmfiles[0]}"
```

Comment:
I'm not sure where this crept in from. Does removing this break a build?

Reply:
The patch I helped you add during one of our 1:1s seems to have disappeared. There should be a patch that removes `--skip-broken --nobes` from this line. I expected to see it and comment that you could squash it with this patch. However, I can not find it on the PR.

Reply:
I am also confused and can't explain where those changes have disappeared ... I removed the `--skip-broken` and `--nobest` parameters again and squashed with the main patch. what is still missing now are the changes to the containerfile for building sambacc during our container build process. I'll try to rediscover them